EventListener/FinishControllerSpanSubscriber.php

```diff
@@ -74,4 +87,20 @@ private function getResponse($event): ?Response
 
         return $response;
     }
+
+    private function addTagsFromStatusCode(?Response $response): void
+    {
+        $responseStatusCode = $response ? $response->getStatusCode() : 'not determined';
+        $this->tracing->setTagOfActiveSpan(HTTP_STATUS_CODE, $responseStatusCode);
+        if ($responseStatusCode && $responseStatusCode >= 400) {
+            $this->tracing->setTagOfActiveSpan(ERROR, true);
+        }
+    }
+
+    private function addTraceIdHeader(?Response $response): void
+    {
+        if ($response && $this->returnTraceId) {
+            $response->headers->set('X-Auxmoney-Opentracing-Trace-Id', $this->tracingId->getAsString());
```

Comment:
$this->tracingId could it be a null pointer exception?

Reply:
No, the container would not build in that case, because it's injected in the constructor.